torch/csrc/autograd/custom_function.cpp

```diff
@@ -337,14 +338,13 @@ static optional_variable_list _process_backward_mode_ad(
         auto& grad_acc = dynamic_cast<AccumulateGrad&>(*grad_acc_fn);
         grad_acc.variable.reset();
       }
-      if (cdata) {
-        impl::rebase_history(var, {cdata, output_nr});
-      }
+      check_inplace(var, true);
```

Comment:
You assume this will never raise right? Because we do more restrictive checks above?
If so, let's leave a small comment about it.

Reply:
Well it does many checks, and will never raise for some of them like leaf mutation indeed.
However, it will raise in cases like checking whether the view is default (which is why we add it here :P)